Reqnroll/Bindings/BindingInvoker.cs

```diff
@@ -41,6 +58,14 @@ public virtual object InvokeBinding(IBinding binding, IContextManager contextMan
         public virtual async Task<object> InvokeBindingAsync(IBinding binding, IContextManager contextManager, object[] arguments, ITestTracer testTracer, DurationHolder durationHolder)
         {
             EnsureReflectionInfo(binding, out _, out var bindingAction);
```

Comment:
This method does some simple, stateless sanity checks. It seemed reasonable to include it in the dry run to further validate the test code's correctness.